{"uuid": "1bca6b6a-1b87-427c-807c-c98f519739ed", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "75d1da27-689a-4e06-b749-bfc6554f15d6", "event": "runner_on_unreachable", "pid": 98249, "created": "2022-05-01T08:37:35.335510", "parent_uuid": "9e73ed77-7fb2-c251-9f81-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "be22758c-87da-4657-921f-2e92143f78af", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-c251-9f81-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-c251-9f81-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-05-01T08:37:35.316373", "end": "2022-05-01T08:37:35.335362", "duration": 0.018989, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "1bca6b6a-1b87-427c-807c-c98f519739ed"}}